apps/vmq_bridge/priv/vmq_bridge.schema

```diff
@@ -265,6 +266,21 @@
          Term
  end}.
 
+{mapping, "vmq_bridge.vmq_metrics_mfa", "vmq_bridge.vmq_metrics_mfa",
```

Comment:
not sure if we really need this to be in the schema...thoughts?

Reply:
adding it the app.src file only isn't enough(I thought it was?) - it's anyway hidden and won't be in the config file.

Reply:
to my knowledge app.src is sufficient... I tested it with my SWC release.

Reply:
You're right - I'd done something wrong when testing. Removed it.